commands/create_test.go

```diff
@@ -20,3 +21,19 @@ func TestValidateSwarmDiscoveryAcceptsValidFormat(t *testing.T) {
 	err := validateSwarmDiscovery("token://deadbeefcafe")
 	assert.NoError(t, err)
 }
+
+func TestRejectCreateTrailingOptions(t *testing.T) {
```

Comment:
This seems to be more of an integration test than a unit test (it tests the whole flow), so my preference would be to leave it out.